src/components/DateTimePicker/DateTimePicker.jsx

```diff
@@ -828,19 +827,17 @@ const DateTimePicker = ({
                         ref={datePickerRef}
                         onChange={onDatePickerChange}
                         onClose={onDatePickerClose}
+                        value={
+                          absoluteValue
+                            ? [
+                                moment(absoluteValue.startDate, dateTimeMask).format('MM/DD/YYYY'),
+                                moment(absoluteValue.endDate, dateTimeMask).format('MM/DD/YYYY'),
```

Comment:
I don't think it's necessary to format on every render. We also do this same format in `onDatePickerChange`, so on non-initial renders I think the format isn't necessary. 

Could we unify this and improve the performance by placing the format inside a useEffect that only runs when the absoluteValue changes? Or maybe a useEffect with an empty dependency array so it's only ran on mount and unmount?

Reply:
Moved the date formatting code to parseDefaultValue() function so that it executes only when the default value is set.